From bff6cdb273874bdcd5edaebbacdcbabcbafd3acf Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Thu, 18 Jun 2020 16:31:15 -0700
Subject: [PATCH 1410/1852] media: hdr-ctrls: Add a missing include

commit 2247d223e3e4ba15020e80c26e4390dcab0c4fe6 from
https://github.com/Xilinx/linux-xlnx.git

The header has data types that have to be included from
types.h.

>> include/media/hdr-ctrls.h:59:2: error: unknown type name '__u8'
      59 |  __u8 eotf;
         |  ^~~~
...

Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Reviewed-by: Dylan Yip <dylan.yip@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/media/hdr-ctrls.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/media/hdr-ctrls.h b/include/media/hdr-ctrls.h
index 5ce1d38c7847..61ca19573cd5 100644
--- a/include/media/hdr-ctrls.h
+++ b/include/media/hdr-ctrls.h
@@ -10,6 +10,8 @@
 #ifndef _HDR_CTRLS_H_
 #define _HDR_CTRLS_H_
 
+#include <linux/types.h>
+
 #define V4L2_CTRL_CLASS_METADATA 0x00b00000
 
 #define V4L2_CID_METADATA_BASE (V4L2_CTRL_CLASS_METADATA | 0x900)
-- 
2.31.1

